src/Pages/Frontpage/Survey/Depricated/Option.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useEffect, useState } from "react";
+import { UserContext } from "../../../../Main/UserContext";
+
+export default (props) => {
```

Comment:
you can use destructuring to make your code more readable, e.g. export default ({text, themeIndex, optionIndex})

Reply:
We are actually doing this in some places. I guess we should be consistent in our format.